src/quo2/components/record_audio/record_audio/__tests__/record_audio_component_spec.cljs

```diff
@@ -0,0 +1,159 @@
+(ns quo2.components.record-audio.record-audio.--tests--.record-audio-component-spec
+  (:require [quo2.components.record-audio.record-audio.view :as record-audio]
+            [status-im.audio.core :as audio]
+            [test-helpers.component :as h]))
+
+(h/describe "record audio component"
+  (h/before-each
+   (fn []
+     (h/use-fake-timers)))
+
+  (h/after-each
+   (fn []
+     (h/clear-all-timers)
+     (h/use-real-timers)))
+
+  (h/test "renders record-audio"
+    (h/render [record-audio/record-audio])
+    (-> (h/expect (h/get-by-test-id "record-audio"))
+        (.toBeTruthy)))
+
+  (h/test "record-audio on-start-recording works"
+    (let [event (js/jest.fn)]
+      (h/render [record-audio/record-audio {:on-start-recording event}])
+      (h/fire-event
+       :on-start-should-set-responder
+       (h/get-by-test-id "record-audio")
+       {:nativeEvent {:locationX 70
+                      :locationY 70}})
+      (-> (h/expect event)
+          (.toHaveBeenCalledTimes 1))))
+
+  (h/test "record-audio on-reviewing-audio works"
+    (let [event (js/jest.fn)]
+      (h/render [record-audio/record-audio {:on-reviewing-audio event}])
+      (with-redefs [audio/start-recording (fn [_ on-start _]
+                                            (on-start))]
+        (h/fire-event
+         :on-start-should-set-responder
+         (h/get-by-test-id "record-audio")
+         {:nativeEvent {:locationX 70
+                        :locationY 70}})
+        (h/advance-timers-by-time 500)
+        (h/fire-event
+         :on-responder-release
+         (h/get-by-test-id "record-audio")
+         {:nativeEvent {:locationX 70
+                        :locationY 70}})
+        (-> (h/expect event)
+            (.toHaveBeenCalledTimes 1)))))
+
+  (h/test "record-audio on-send works after reviewing audio"
+    (let [event (js/jest.fn)]
+      (h/render [record-audio/record-audio {:on-send event}])
+      (with-redefs [audio/start-recording        (fn [_ on-start _]
+                                                   (on-start))
+                    audio/get-recorder-file-path (fn [] "audio-file-path")]
+        (h/fire-event
+         :on-start-should-set-responder
+         (h/get-by-test-id "record-audio")
```

Comment:
You can safely use keywords instead of strings. The functions in `test-helpers.component` all make sure keywords are converted back to strings.

Reply:
Good to know, I saw some other tests using strings so I wasn't sure, but I will change it